### PR TITLE
updating WDL to avoid empty shards and also out of memory errors

### PR DESCRIPTION
**Pull request by kcibul**

### CODE REVIEW

File: extract/raw_array_cohort_extract.wdl [hunk 1]
```diff
@@ -169,10 +171,10 @@ task ExtractTask {
                 -O "~{output_file}" \
                 ~{probe_info_clause} \
                 --project-id "~{project_id}" \
-                --cohort-sample-table "~{fq_cohort_mapping_table}" \
+                --sample-info-table "~{fq_cohort_mapping_table}" \
```
Comment (ahaessly) [inline extract/raw_array_cohort_extract.wdl]: did you rebase off of my changes? i think i changed the name to cohort-sample-table. unless it got changed back

**Review by ahaessly — APPROVED** [review]
on param name to check

**Review by meganshand — APPROVED** [review]
Are you planning on changing GATK to not blow up if there's no data (in addition to the changes here which look good)?

**Comment by kcibul** [comment]
I'll make a ticket for that GATK change @meganshand -- it's not clear to me how we would do that nicely... but we should figure that out

File: extract/raw_array_cohort_extract.wdl (hunk 1 above)
Comment (kcibul) [reply extract/raw_array_cohort_extract.wdl]: rebased

**Pull request merged**